pkgs/by-name/go/gophish/package.nix

```diff
@@ -0,0 +1,28 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+}: buildGoModule rec {
```

Comment:
```suggestion
}: 
buildGoModule rec {
```

Reply:
fixed, thank you for the review! :) if there's absolutely anything else then please lmk